### PR TITLE
First step toward end-to-end test

### PR DESCRIPTION
**Pull request by blp**
This adds an example program to the `dbsp_nexmark` crate to generate Nexmark input data into Kafka.

Is this a user-visible change (yes/no): no

### CODE REVIEW

File: crates/nexmark/src/config.rs [hunk 1]
```diff
@@ -90,40 +126,11 @@ pub struct Config {
     /// Specify the proportion of events that will be new people.
     #[clap(long, default_value = "1", env = "NEXMARK_PERSON_PROPORTION")]
     pub person_proportion: usize,
-
-    /// Dump DBSP profiles for all executed queries to the specified directory.
-    #[clap(long, env = "NEXMARK_PROFILE_PATH")]
```
Comment (mihaibudiu) [inline crates/nexmark/src/config.rs:95]: these clap things are not important?

File: crates/nexmark/examples/generate.rs [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        }
+    }
+
+    /// Writes `record`, flushing automatically if the threshold is crossed.
+    pub fn write<S>(&mut self, record: S)
+    where
+        S: Serialize,
+    {
+        self.writer.serialize(record).unwrap();
+        self.flush_if(512 * 1024);
```
Comment (mihaibudiu) [inline crates/nexmark/examples/generate.rs:40]: should this be a more visible constant?

**Review by mihaibudiu — APPROVED** [review]

File: crates/nexmark/src/config.rs (hunk 1 above)
Comment (blp) [reply crates/nexmark/src/config.rs:95]: I moved them to another struct. Nothing was deleted (unless I made a mistake).

Comment (mihaibudiu) [reply crates/nexmark/src/config.rs:95]: found them

File: crates/nexmark/examples/generate.rs (hunk 2 above)
Comment (blp) [reply crates/nexmark/examples/generate.rs]: I added a `MAX_LEN` constant and made the threshold more precise.

**Pull request merged**